mozilla/bedrock
# Add QR Code generator template helper function

Re #9875

## Testing

You can try it locally by simply installing the new requirements and adding a call to a template:

1. `pip install -r requirements/base.txt`
2. Edit a template and add a call to the thing: `{{ qrcode('https://bedrock.rtfd.org') }}`
3. Run the local server and see the QR Code on the page

If you're wanting to use the Docker setup then you'll need to build new images `make build` to get the new deps.
